docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,348 @@
+
+    >microk8s.kubectl wait -n ingress --for=condition=Ready pod -l name=nginx-ingress-microk8s --timeout=300s
+
+4. Export your microk8s cluster configuration to the local kubeconfig file
+    >microk8s.config > ~/.kube/config
+
+After runing these commands, we have ensured that dns, http, and nginx ingress are enabled and working properly inside the cluster.
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
+
+First we must install `helm` onto the cluster. With `helm`, we can easily install charts and manage the lifecycle of the installed artifacts. One example is the ability to rollback a failed install.
+
+1. Assuming you would want to install `helm` version 2.16.3, export the following environment variable
+    >export HELM_VERSION=2.16.3
+   1. Please note that we currently do *not* support helm v3.
+2. Run this script that will retrieve the desired `helm` version and install it onto the machine's `/usr/local/bin`:
+   1. Define a HELM_PKG variable
+      >HELM_PKG=helm-v${HELM_VERSION}-linux-amd64.tar.gz
+   2. Get the package
+      >wget https://get.helm.sh/$HELM_PKG
+   3. Extract the tarball
+      >tar xvzf $HELM_PKG
+   4. Move the binaries to `/usr/local/bin`
+      >mv linux-amd64/tiller linux-amd64/helm /usr/local/bin
+   5. Delete the extracted tarball directory
+      >rm -rvf $HELM_PKG linux-amd64/
+
+3. We must now deploy `tiller` onto the cluster.
+   1. First we must set a `serviceaccount` and a `clusterrolebinding` for `tiller` on the microk8s cluster:
+      >microk8s.kubectl create serviceaccount --namespace kube-system tiller
+
+      >microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+   1. Then we can install `tiller`, which will be installed with the same `HELM_VERSION` that you have specified earlier:
+      >helm init --service-account tiller
+   1. We wait for tiller to be ready
+      >microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s
+
+### 2. Codacy chart
+
+Please refer to [our chart installation documentation](https://github.com/codacy/chart/blob/master/docs/install.md).
+
+Additionally, any `kubectl` command from [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md) must be executed as a `microk8s.kubectl` command.
+
+#### 1. Default Values
+
+We provide a base (values file)[##microk8s-values.yaml] that you can use for your microk8s Codacy installation. This file provisions a smaller number of component replicas with limited resources (i.e. CPU and memory). Should you want to use this as a template, please pass your `microk8s-values.yaml` file on (4.) of [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md). Lastly, it is important to correctly set your `codacy-api.ingress.hosts.host` if you are using this file as a template.
```

Comment:
Consider providing the `.yaml` file as a standalone file in the GitHub repository just like we did for AKS and EKS:

https://github.com/codacy/chart/tree/master/docs/infrastructure/AKS
https://github.com/codacy/chart/tree/master/docs/infrastructure/EKS

Reply:
Sorry, I confused the files. The ones in the link above are the Terraform backend files and have nothing to do with the `microk8s-values.yaml` file mentioned here.

The file I wanted to mention is actually this one:
https://github.com/codacy/chart/blob/master/codacy/values-production.yaml